### ISSUE TITLE
Unable to delete related work

### ISSUE THREAD

**Issue by mariapraetzellis** [CDLUC3/dmptool#372]
Bug: Unable to delete related work line using the close icon with both mouse and keyboard.

We should discuss how we want this work since we'll also need to remove the related works from the metadata sent to DataCite & I am not sure this is feasible.
